syft/artifact/id.go

```diff
@@ -13,7 +13,7 @@ type Identifiable interface {
 	ID() ID
 }
 
-func IDFromHash(obj interface{}) (ID, error) {
+func IDByHash(obj interface{}) (ID, error) {
```

Comment:
Nice I also thought this was a little awkward when we were debugging this AM.

Reply:
yeah, I realized I almost got the name right, so was a good time to tweak it